### ISSUE TITLE
Unable to replicate published benchmark accuracy on cifar10 dataset

### ISSUE THREAD

**Issue by abdul53saifee** [google-research/big_transfer#59]
I tried to finetune big transfer code for cifar 10 dataset using the colab provided in [TF Blog](https://blog.tensorflow.org/2020/05/bigtransfer-bit-state-of-art-transfer-learning-computer-vision.html)

The dataset was split on 90% training and 10% validation.
The steps per epochs were modified as per dataset number of samples.
Bit hyper rule were kept as it is without any modification
Model used for finetuning was Resnet-m-50x1 and Resnet-m-101x3

Accuracy of model reached maximum of 94.3% for resnet-m-50x1 against the mentioned accuracy of 97.2
Accuracy for resnet-m-101x3 was reaching max of 93.2 against the mentioned accuracy of 98.5 and no further improvement was observed even running for some more epochs.

Accuracy source :- [README.md](https://github.com/google-research/big_transfer/blob/master/README.md)
